xml/remote_administration.xml

```diff
@@ -258,16 +258,15 @@ nodenames[A-B]          attr[=value],attr[=value],...
   <para>
    <emphasis>MUNGE</emphasis> allows for secure communications between
    different machines which share the same secret key. The most common
-   usecase is the <emphasis>Slurm</emphasis> workload manager, which
+   use case is the <emphasis>Slurm</emphasis> workload manager, which
```

Comment:
It is quite common practice to emphasise the first occurrence of a new term, just in case it has some other usage that the reader may know, in order to tell them this is a new usage.

I have removed emphasis/literal etc. from subsequent usages already.

I have added all 3 entities you suggest, & I have changed Slurm and MUNGE into entities, but I am hesitant to also do mrsh because of the difficulty of distinguishing its name from an invocation.